src/Sylius/Bundle/AdminBundle/Resources/views/PromotionCoupon/BreadCrumb/updateBreadCrumb.html.twig

```diff
@@ -0,0 +1,12 @@
+{% import 'SyliusAdminBundle:Crud/Macro:breadCrumb.html.twig' as breadCrumb %}
+
+{% set crumbs = {
```

Comment:
You don't need to index elements manually, just use `[]` instead of `{}` eg. `{% set breadcrumbs = ['0th element', '1st element'] %]`